luigi/mock.py

```diff
@@ -170,6 +171,22 @@ def skip(func):
     """
     Sort of a substitute for unittest.skip*, which is 2.7+.
     """
+    if inspect.isclass(func):
+        return object
+
     def wrapper():
         pass
+
     return wrapper
+
+
+def skipIf(condition):
+    """
+
+    """
+    if condition:
+        return skip
+    return lambda x: x
+
+
+skipIfPy3k = skipIf(six.PY3)
```

Comment:
Yeah I think that makes sense. But if you are busy we can merge now and fix it later.